arrow/src/compute/kernels/temporal.rs

```diff
@@ -338,7 +338,7 @@ where
                 scratch
             )
         }
-        dt => return_compute_error_with!("weekday does not support", dt),
+        dt => return_compute_error_with!("weekday0 does not support", dt),
```

Comment:
This function is now called `num_days_from_sunday`
```suggestion
        dt => return_compute_error_with!("num_days_from_sunday does not support", dt),
```

Reply:
Ty